app/_src/kic-v2/guides/prometheus-grafana.md

```diff
@@ -2,115 +2,81 @@
 title: Integrate the Kong Ingress Controller with Prometheus/Grafana
 ---
 
-The {{site.kic_product_name}} can give you visibility into how {{site.base_gateway}} is performing and how the services in your Kubernetes cluster are responding to the inbound traffic.
+The {{site.kic_product_name}} gives you visibility into how {{site.base_gateway}} is performing and how the services in your Kubernetes cluster are responding to the inbound traffic.
 
-This guide walks you through setting up monitoring for
-{{site.base_gateway}} with Prometheus.
+Learn to set up monitoring for {{site.base_gateway}} with Prometheus.
 
-{:.important}
-> As of {{site.kic_product_name}} 2.0, there are additional
-> performance metrics associated with the configuration process
-> (as opposed to the runtime performance of the Gateway), described in detail
-> in the [Prometheus metrics reference](/kubernetes-ingress-controller/{{page.kong_version}}/references/prometheus/).
+As of {{site.kic_product_name}} 2.0, there are additional
+performance metrics associated with the configuration process
+rather than the runtime performance of the Gateway. For more information, see the [Prometheus metrics reference](/kubernetes-ingress-controller/{{page.kong_version}}/references/prometheus/).
 
-This guide was originally posted on [Kong blog](https://konghq.com/blog/observability-kubernetes-kong/).
-
-## Prerequisites
-
-You’ll need a few things before we can start:
-
-- **Kubernetes cluster**: You can use Minikube or a GKE cluster for the
-  purpose of this tutorial. We are running a GKE Kubernetes cluster v1.12.x.
-- **Helm**: We will be using [Helm](https://helm.sh/)
-  to install all of our components.
-  Tiller should be installed on your Kubernetes cluster and
-  Helm CLI should be available on your workstation.
-  You can follow Helm’s quickstart guide to set up helm.
-
-Once you have Kubernetes and Helm set up, please proceed.
-
-Caution: Settings here are tweaked to keep this guide simple.
-These settings are not meant for production usage.
+{% include_cached /md/kic/prerequisites.md kong_version=page.kong_version disable_gateway_api=true %}
```

Comment:
@mheap  in the deploy preview the results appear. I think this was fixed in one the PRs that you merged yesterday. This is just an FYI.